### PR TITLE
Add vaxrank unit tests

### PR DESCRIPTION
**Pull request by julia326**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -55,6 +57,11 @@ before_script:
 script:
   - ./lint.sh
   - nosetests -sv test --with-coverage --cover-package=mhctools
+  # run vaxrank unit tests as a general integration test to make sure the stack isn't broken
+  - pip install -r $PWD/vaxrank/requirements.txt
+  - pyensembl install --release 87 --species human
+  - pyensembl install --release 87 --species mouse
+  - travis_wait nosetests test -a '!skip' $PWD/vaxrank
```
Comment (tavinathanson) [inline .travis.yml]: Have had issues with `travis_wait` not doing what it says it does in the past; what is the goal here?

Comment (tavinathanson) [reply .travis.yml]: And what is the `!skip` doing? Can you comment that?

**Review by tavinathanson — COMMENTED** [review]
LGTM, some minor comments

**Review by tavinathanson — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -49,18 +52,25 @@ install:
...
   - ./lint.sh
-  - nosetests -sv test --with-coverage --cover-package=mhctools
+  # -a '!skip' lets us skip certain tests on travis (relevant for vaxrank)
+  # this command runs both mhctools and vaxrank tests
+  - nosetests -sv test --with-coverage --cover-package=mhctools -a '!skip' $PWD/vaxrank
 after_success:
   coveralls
 deploy:
   provider: pypi
+  distributions: sdist
```
Comment (tavinathanson) [inline .travis.yml:72]: What does this do?

File: .travis.yml [hunk 3]
```diff
@@ -49,18 +52,25 @@ install:
...
   user: hammerlab
   password: # See http://docs.travis-ci.com/user/encryption-keys/
-    secure: "KTo/ea1UJ4fSjXrK7YxHZSBlpEtx6zONzwRgIX8Ec3DRQmlhskIky8YD6nrmH4yjm7pNJOx0DNF2x9kF1qAXLjlLNWBp1V6dk+nSEmyW0MfX08Kw2dEn9N1+P0O9EcVUqnMXO7I8JSG8GDkBYhngwMDUCsm1YDHWU9f7tx/se2k="
+    secure: "hptQUnWnbqfMeiDPWX5Jw34qBHfuDuKofjyeF6m+eofg5XRWr4HsKZraghk7lMLQWcmkpX9FPwMYXHfQA+FO+81ColrPll3BCx7DlG71qGOoNEsbVN8C/OM6z+/+QWXlebxHGu9f06uFJC8glo8gKUSKrtNjJ5GsI+buDyE2a74="
```
Comment (tavinathanson) [inline .travis.yml]: What changed?

**Review by tavinathanson — COMMENTED** [review]
A few minor comments

File: .travis.yml (hunk 2 above)
Comment (julia326) [reply .travis.yml:72]: modeling this one on topiary - that travis config is set up to auto-deploy to PyPi on an __init__.py version bump, and this is the only difference

File: .travis.yml (hunk 3 above)
Comment (julia326) [reply .travis.yml]: that was the output when I ran `travis encrypt <the hammerlab password string>` just now

Comment (tavinathanson) [reply .travis.yml]: Was it not working before? This time around did you still use `-r mhctools`?

File: .travis.yml (hunk 2 above)
Comment (tavinathanson) [reply .travis.yml:72]: Not sure how this relates to that https://docs.travis-ci.com/user/deployment/pypi/#Uploading-different-distributions

But fine with me

File: .travis.yml [hunk 4]
```diff
@@ -49,18 +52,26 @@ install:
   - pip install -r requirements.txt
   - pip install .
   - pip install coveralls
+  # install vaxrank and its dependent data
+  - pip install --upgrade -r $PWD/vaxrank/requirements.txt
+  - pyensembl install --release 87 --species human
```
Comment (iskandr) [inline .travis.yml:57]: @tavinathanson @julia326 Thoughts on how to shrink the footprint of Ensembl release installs? I was thinking about adding a pyensembl option to get rid of noncoding transcripts

**Review by iskandr — APPROVED** [review]

File: .travis.yml (hunk 4 above)
Comment (julia326) [reply .travis.yml:57]: @iskandr I have a vague memory of you caching the Ensembl data in other Travis configs, did that ever work out?

**Pull request merged**